Docs/Book/GettingStartedWithContributing.md

```diff
@@ -0,0 +1,633 @@
+      - [RDKit blog](https://greglandrum.github.io/rdkit-blog/posts/2020-03-30-setting-up-an-environment.html)
+      - [Practical Chemoinformatics](https://practicalcheminformatics.blogspot.com/)
+      - [Is life worth living?](https://iwatobipen.wordpress.com/)
+      - [Mac in Chem](https://macinchem.org/?s=rdkit)
+      - [Cheminformania](https://www.cheminformania.com/?s=rdkit)
+  - From UGM talks (the relevant repositories are listed in the README.md on GitHub)
+  - Translation to other languages (e.g. see the [Japanese](https://www.rdkit.org/docs_jp/The_RDKit_Book_jp.html) copy of the RDKit Book)
+  - To this guide if instructions need updated!
+
+### A Whistle Stop Tour of the RDKit Documentation
+
+The RDKit documentation consists of several different sections. Here we'll give an overview of how each of these is generated, and what contributions should go where. An index of the RDKit documentation can be found [here](https://www.rdkit.org/docs/index.html). The major parts of the documentation (excluding the Overview and Installation instructions, which generally won't require community contributions) are as follows: 
+- The [Python API](https://www.rdkit.org/docs/api-docs.html) and [C++ API](https://www.rdkit.org/docs/cppapi/index.html) docs are generated automatically from the codebase.
+- The "Getting Started With" guides for [Python](https://www.rdkit.org/docs/GettingStartedInPython.html) and [C++](https://www.rdkit.org/docs/GettingStartedInC%2B%2B.html) provide an overview of how to use the core functionality of RDKit. 
+- The [RDKit Cookbook](https://www.rdkit.org/docs/Cookbook.html) contains more in-depth example recipes in Python for how to use RDKit for common tasks, generally contributed by the community.
+- The [RDKit Book](https://www.rdkit.org/docs/RDKit_Book.html) gives more technical details about how various parts of the RDKit work.  
+- Not strictly speaking documentation but we'll include it here for completeness, Greg writes a [Blog](https://greglandrum.github.io/rdkit-blog/) highlighting in more detail how to use new and existing bits of the RDKit.
+
+### How to Write RDKit Documentation
+
+The RDKit documentation is written mostly in [reStructuredText](https://docutils.sourceforge.io/rst.html). Full tutorials can be found online, but a brief overview will be given here to get you started. 
+
+*Headings*
+
+  ```
+  This is a TITLE
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  This is a HEADING 
+  **************
+
+  This is a SUBHEADING
+  ========================
+  ```
+
+*Code Examples*
+
+The RDKit docs support inclusion of Sphinx doctests to allow the result of a code example to appear in the text. 
+
+  ```
+  .. doctest::
+
+    >>> mol = Chem.MolFromSmiles('CO(C)C')
+    >>> print(print(mol.GetNumAtoms()))
+    4
+  ```
+
+*Hyperlinks*
+
+  ```
+  `RDKit Website`_
+
+  .. _RDKit Website: https://www.rdkit.org/
+  ```
+
+*Images*
+
+  ```
+  .. image:: pathtoimage/name.png
+  ```
+
+### Tutorials
```

Comment:
The tutorials repo seems moribund - no updates since last year - so it's probably better to just drop this section.

Reply:
That's a shame - I think mention of them somewhere is still useful for a "beginners" guide, will add a link beside the description of the cookbook above.